pkg/controller/statefulset/stateful_set_test.go

```diff
@@ -46,7 +46,7 @@ func TestStatefulSetControllerCreates(t *testing.T) {
 		set = obj.(*apps.StatefulSet)
 	}
 	if set.Status.Replicas != 3 {
-		t.Error("Falied to scale statefulset to 3 replicas")
+		t.Error("Failed to scale statefulset to 3 replicas")
```

Comment:
Ideally these should be changed to match the conventional Go style:

https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures

```go
if got, want := set.Status.Replicas, 3; got != want {
  t.Errorf("set.Status.Replicas = %v; want %v", got, want)
}
```

Would you be willing to update the PR to that form?